code/modules/mob/living/carbon/human/species.dm

```diff
@@ -1892,6 +1893,110 @@ GLOBAL_LIST_EMPTY(roundstart_races)
+						humi.adjust_bodytemperature(max(gas_amount * 0.25, BODYTEMP_HEATING_MAX) * delta_time)
+					continue
+				if(gas_amount > 5 && gas_amount < 10)
+					to_chat(humi, span_warning("[pick("Your skin feels cold.", "You feel cold.")]"))
+				else
+					to_chat(humi, span_warning("[pick("Your skin starts to freeze!", "You feel very cold!")]"))
+					var/current_stamina = humi.getStaminaLoss()
+					if(current_stamina > 25)
+						humi.adjustStaminaLoss(gas_amount * 0.15 * delta_time)
+					humi.adjust_bodytemperature(max(- gas_amount * 0.25, BODYTEMP_COOLING_MAX) * delta_time)
+					if(humi.bodytemperature < 265)
+						humi.apply_status_effect(/datum/status_effect/freon)
+			if(/datum/gas/halon)
+				if(gas_amount > 15)
+					humi.apply_status_effect(/datum/status_effect/stoned)
+			if(/datum/gas/healium)
+				if(gas_amount > 5)
+					to_chat(humi, span_notice("You feel better."))
+					humi.adjustBruteLoss(- gas_amount * 0.5 * delta_time)
+					humi.adjustFireLoss(- gas_amount * 0.15 * delta_time)
+					humi.adjustStaminaLoss(- gas_amount * 0.15 * delta_time)
+			if(/datum/gas/helium)
+				if(gas_amount > 15)
+					humi.AddElement(/datum/element/forced_gravity, 0)
+					addtimer(CALLBACK(humi, .proc/_RemoveElement, list(/datum/element/forced_gravity, 0)), 1 SECONDS * delta_time)
+			if(/datum/gas/hypernoblium)
+				if(gas_amount > 5)
+					humi.apply_status_effect(/datum/status_effect/antimagic)
+			if(/datum/gas/nitryl)
+				if(ispodperson(humi))
+					humi.adjustToxLoss(- gas_amount * 0.5 * delta_time)
+					humi.adjustBruteLoss(- gas_amount * 0.5 * delta_time)
+					var/current_reagent = humi.reagents.get_reagent_amount(/datum/reagent/nitryl)
+					humi.reagents.add_reagent(/datum/reagent/nitryl, max(0, 1 - current_reagent))
+					continue
+				if(gas_amount > 5 && gas_amount < 10)
+					to_chat(humi, span_warning("[pick("Your skin feels itchy.", "You feel like burning.")]"))
+				else
+					to_chat(humi, span_warning("[pick("Your skin feels hot.", "You feel like burning.")]"))
+					humi.adjustFireLoss(gas_amount * 0.5 * delta_time)
+			if(/datum/gas/oxygen)
+				if(gas_amount > 50 && environment.temperature > 400)
+					humi.adjust_fire_stacks(gas_amount * 0.1)
+			if(/datum/gas/plasma)
+				if(isplasmaman(humi))
+					humi.adjustBruteLoss(- gas_amount * 0.5 * delta_time)
+					humi.adjustFireLoss(- gas_amount * 0.15 * delta_time)
+					humi.adjustStaminaLoss(- gas_amount * 0.15 * delta_time)
+					continue
+				if(gas_amount > 5)
+					to_chat(humi, span_warning("[pick("Your skin stings.", "You feel like there is something oily on you.", "You feel something crawling on you!", "Your liver starts moving!", "Your right eye seems sad.", "Your left leg want to be right.")]"))
+					humi.hallucination += gas_amount * 0.5 * delta_time
+					humi.hallucination = clamp(humi.hallucination, 0, 100)
+			if(/datum/gas/pluoxium)
+				if(gas_amount > 10)
+					humi.adjustOxyLoss(- gas_amount * 0.15 * delta_time)
```

Comment:
I'm more ok with this because it's got soul, and oxy loss will heal on its own most of the time. not sure it'll ever see use because of this, but I can't call it overpowered

Reply:
yeah i added it because it seemed fun tbh